debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSnapshotChangeEventSource.java

```diff
@@ -158,7 +158,7 @@ private boolean areSameTimestamp(Long scn1, long scn2) throws SQLException {
             return Optional.empty();
         }
 
-        StringBuilder lastDdlScnQuery = new StringBuilder("SELECT MAX(TIMESTAMP_TO_SCN(last_ddl_time))")
+        StringBuilder lastDdlScnQuery = new StringBuilder("SELECT TIMESTAMP_TO_SCN(MAX(last_ddl_time))")
```

Comment:
@jpechane, any thoughts?

Reply:
@shelfoo Looks fine for me, is there a way how to test the change?

Reply:
@jpechane I can maybe get a green->green test in place, tough to do red->green without having a db that is old. Other than that just running this sql on any db proves that it has the same effect

Reply:
@shelfoo ok, let's do it without test. Thanks a lot, merging.